### ISSUE TITLE
Standardize model validation output and metrics

### ISSUE THREAD

**Issue by Cardiobotics** [Eiphodos/SUEF#12]
Currently, model output and metrics are different depending on the loss function (classification/regression/ordinal regression). We need to fix this for the validation data (not training data).

Regardless of loss/problem type, validation output should be placed in the following bins with this rule:
"För alla värden på EF där EF % 5 != 0 så har EF satts till närmaste tal delbart med 5.
Exempel 1: Orginal EF = 32, nytt EF = 30.
Exempel 2: Orginal EF = 38, nytt EF = 40.
Alla värden på EF där EF < 20 har ändrats till 20.
Exempel: Orginal EF = 15, nytt EF = 20.
Alla värden på EF där EF > 70 har ändrats till 70. 
Exempel: Orginal EF = 85, nytt EF = 70"

Thereafter the following metrics should be computed and logged to neptune:

[computed on EF values]
val/r2
val/best_r2

[computed on class bins]
val/top1_accuracy
val/best_top1_accuracy

**Comment by jenniferalven** [comment]
Add two more metrics:

[computed on EF values]
val/mean_ae
val/best_mean_ae
val/median_ae
val/best_median_ae
